pennylane/beta/qnode.py

```diff
@@ -28,6 +28,35 @@
 class QNode:
     """Represents a quantum node in the hybrid computational graph.
 
+    .. warning::
+
+        This QNode is a beta feature. It differs from the standard :class:`~.pennylane.QNode`
+        in several ways:
+
+        - Custom gradient transforms can be specified as the differentiation method.
+
+        - Arbitrary :math:`n`-th order derivatives are supported on hardware using
+          gradient transforms such as the parameter-shift rule. To specify that an :math:`n`-th
+          order derivative of a QNode will be computed, the ``max_diff`` argument should be set.
+          By default, this is set to 1 (first-order derivatives only).
+
+        - Internally, if multiple circuits are generated for execution simultaneasouly, they
+          will be packaged into a single job for execution on the device. This can lead to
+          significant performance improvement when executing the QNode on remote
+          quantum hardware.
+
+        In an upcoming release, this QNode will replace the existing one. If you come across any
+        bugs while using this QNode, please let us know via a `bug report <https://github.com/PennyLaneAI/pennylane/issues/new?assignees=&labels=bug+%3Abug%3A&template=bug_report.yml&title=%5BBUG%5D>`__
+        on our GitHub bug tracker.
+
+        Currently, this beta QNode does not support the following features:
+
+        - Circuit decompositions
+        - Non-mutabability via the `mutable` keyword argument

```

Comment:
```suggestion
        - Non-mutability via the `mutable` keyword argument
```
This is a funny-sounding typo :laughing: